doc/modules/classes.rst

```diff
@@ -1197,7 +1197,7 @@ See the :ref:`metrics` section of the user guide for further details.
    preprocessing.MaxAbsScaler
    preprocessing.MinMaxScaler
    preprocessing.Normalizer
-   preprocessing.OneHotEncoder
```

Comment:
This indicated deprecation of the old class? I'm not sure we want to do that yet.